src/ProjectService.Business/Commands/Project/GetProjectCommand.cs

```diff
@@ -69,30 +69,29 @@ private DepartmentInfo GetDepartment(Guid departmentId, List<string> errors)
             return null;
         }
 
-        private List<ImageInfo> GetImages(List<Guid> imageIds, List<string> errors)
+        private List<ImageInfo> GetUserAvatars(List<Guid> imageIds, List<string> errors)
         {
             if (imageIds == null || imageIds.Count == 0)
             {
-                return new();
+                return null;
             }
 
             string errorMessage = "Can not get images. Please try again later.";
-            const string logMessage = "Errors while getting images with ids: {Ids}.";
+            string logMessage = "Errors while getting images with ids: {Ids}. Errors: {Errors}";
 
             try
             {
                 IOperationResult<IGetImagesResponse> response = _rcImages.GetResponse<IOperationResult<IGetImagesResponse>>(
-                    IGetImagesRequest.CreateObj(imageIds)).Result.Message;
+                    IGetImagesRequest.CreateObj(imageIds, ImageSource.User)).Result.Message;
 
-                if (response.IsSuccess)
+                if (response.IsSuccess && response.Body != null)
```

Comment:
IGetImagesResponse.CreateObj (see LT.DigitalOffice.Models.Broker.Responses.Image) body cannot be null

Reply:
done